test/unit/test_disco_deploy.py

```diff
@@ -1133,7 +1242,7 @@ def test_run_integration_tests_success(self):
         self._ci_deploy._get_old_instances = MagicMock(return_value=[inst1])
         self._ci_deploy._get_new_instances = MagicMock(return_value=[inst2])
         self.assertIsNone(self._ci_deploy.test_ami(ami, dry_run=False))
-        self._ci_deploy._get_old_instances.assert_called_with(ami.id)
+        self._ci_deploy._get_old_instances.assert_called_with(ami.id, launch_time=ANY)
```

Comment:
I much prefer the specificity of saying `launch_time=ANY` as opposed to just `ANY`. Can we do that for all of these calls?